README.md

```diff
@@ -1,118 +1,360 @@
 # Redis on Unikraft
 
-This application starts a Redis server.
+This application starts an Redis web server with Unikraft.
+Follow the instructions below to set up, configure, build and run Redis.
 
-To configure, build and run the application you need to have [kraft](https://github.com/unikraft/kraft) installed.
+## Requirements
 
-To be able to interact with the server, configure the application to run on the KVM platform:
-```
-$ kraft configure -p kvm -m x86_64
-```
+In order to set up, configure, build and run Redis on Unikraft, the following packages are required:
 
-Build the application:
-```
-$ kraft build
-```
+* `build-essential` / `base-devel` / `@development-tools` (the meta-package that includes `make`, `gcc` and other development-related packages)
+* `flex`
+* `bison`
+* `git`
+* `wget`
+* `uuid-runtime`
+* `qemu-system-x86`
+* `qemu-kvm`
+* `sgabios`
+* `gcc-aarch64-linux-gnu`
 
-We use a virtual bridge to create a connection between the VM and the host system.
-We assign address `172.44.0.1/24` to the bridge interface (pointing to the host) and we assign address `172.44.0.2/24` to the virtual machine, by passing boot arguments.
-The IP addresses are of our choosing, they can be changed to other values.
+GCC >= 8 is required to build Redis on Unikraft.
 
-We run the commands below to create and assign the IP address to the bridge `virbr0` (once again, our choice of name; any name works):
-```
-$ sudo brctl addbr virbr0
-$ sudo ip a a  172.44.0.1/24 dev virbr0
-$ sudo ip l set dev virbr0 up
+On Ubuntu/Debian or other `apt`-based distributions, run the following command to install the requirements:
+
+```console
+$ apt-get install -y --no-install-recommends \
+  build-essential \
+  gcc-aarch64-linux-gnu \
+  libncurses-dev \
+  libyaml-dev \
+  flex \
+  bison \
+  git \
+  wget \
+  uuid-runtime \
+  qemu-kvm \
+  qemu-system-x86 \
+  sgabios
 ```
```

Comment:
Add instructions on configuring bridge networking for QEMU, similar to [the instructions for `app-nginx`](https://github.com/unikraft/app-nginx#requirements).